src/elements/content-sidebar/MetadataSidebarRedesign.scss

```diff
@@ -4,7 +4,6 @@
     border-left: 1px solid tokens.$gray-10;
 
     .bcs-MetadataSidebarRedesign-content {
-        position: absolute;
```

Comment:
Why did we need to remove this?

Reply:
We don't need it. I was testing sth ad forgot to bring it back. Thanks for noticing! I am getting rid of this change.